src/main/java/org/geysermc/resources/state/type/CampfireExtinguishedMapper.java

```diff
@@ -0,0 +1,14 @@
+package org.geysermc.resources.state.type;
+
+import org.geysermc.resources.Pair;
+import org.geysermc.resources.state.StateMapper;
+import org.geysermc.resources.state.StateRemapper;
+
+@StateRemapper(value = "lit", blockRegex = ".*campfire.?$")
+public class CampfireExtinguishedMapper extends StateMapper<Boolean> {

```

Comment:
Needs to be named to CampfireLitMapper. Naming convention is `(block)(java state)Mapper`.